uber/templates/emails/emails_by_kickin.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}{% set admin_area=True %}
+{% block title %}Email addresses by kick-in{% endblock %}
+{% block content %}
+
+  <h1>Emails by Kick-in Level</h1>
+
+  Use this page to generate a CSV file of emails of people who donated at a certain kick-in level at
+  {{ c.EVENT_NAME }}.<br/><br/>
+
+  <form action="emails_by_kickin_csv">
+    Generate list of emails from attendees with one of the following kick-in levels or more:<br/>
+    <select name="amount_extra">
+      {{ options(c.DONATION_TIER_OPTS) }}
```

Comment:
One thing to potentially keep in mind, though I'm not sure how big a deal this is: since $0 is the "No Thanks" tier, this form allows someone to generate a CSV containing all the names and email addresses of practically everyone with an attendee badge.  More specifically, someone with only ``c.PEOPLE`` access.

I know we have some generic import/export features which already do the same thing, though I think those are a little more locked down.

Dunno if that's a big deal, I'm probably just being paranoid.  Also there's probably some other place where we're doing that already anyway, in which case there's no reason to worry about it here specifically.